includes/functions.php

```diff
@@ -1549,24 +1549,33 @@ function follow( $remote_actor, $user_id ) {
 		return $remote_actor_post;
 	}
 
-	$actor = Actors::get_by_id( $user_id );
+	return Following::follow( $remote_actor_post, $user_id );
+}
 
-	if ( \is_wp_error( $actor ) ) {
-		return $actor;
+/**
+ * Unfollow a user.
+ *
+ * @param string $remote_actor The Actor URL or WebFinger Resource.
+ * @param int    $user_id      The ID of the WordPress User.
+ *
+ * @return \WP_Post|\WP_Error The ID of the Outbox item or a WP_Error.
+ */
+function unfollow( $remote_actor, $user_id ) {
```

Comment:
How would you feel about waiting to add this function until it's necessary and has a use case? This would be the fifth unused function in `functions.php`.